docs/reference/ml/df-analytics/apis/put-dfanalytics.asciidoc

```diff
@@ -164,27 +168,27 @@ The configuration information necessary to perform
 [%collapsible%open]
 =====
 `compute_feature_influence`::::
-(Optional, boolean) 
+(Optional, boolean)
```

Comment:
Just wondering (as my Intellij also removes trailing whitespace).
Should we piggy back the whitespace changes with our PRs or maybe there should be a separate PR to clean up the whitespace so that it is easier to review the "real" changes?

Reply:
It is weird that trailing white spaces are there to begin with. I am not sure which is best. But backing out only the white space changes in a file would take some work/scripting.